lib/manageiq/release/internationalization.rb

```diff
@@ -0,0 +1,104 @@
+module ManageIQ
+  module Release
+    class Internationalization
+
+      def self.instance(options = {})
+        new(options[:branch], :dry_run => options[:dry_run])
+      end
+
+      def self.update_pot(options = {})
+        instance(options).update_pot
+      end
+
+      attr_reader :branch, :dry_run
+
+      def initialize(branch, dry_run: true)
+        @branch  = branch || 'master'
+        @dry_run = dry_run
+      end
+
+      #
+      # In ManageIQ/manageiq repo, run equivalent of `rake locale:update`
+      # If there are material differences, push up to GitHub
+      def update_pot
+        generate_pot
+      end
+
+      private
+
+      def execute(command, description = nil)
+        description ||= "*** Running `#{command}` ***"
+        puts description
+        system(command)
+      end
+
+      def create_database_yml
+        puts "*** Creating config/database.yml ***"
+        database_config = YAML.load_file("config/database.pg.yml")
+        unless database_config.key?('i18n')
+          database_config['i18n'] = database_config['test'].dup
+          database_config['i18n']['database'] = "vmdb_i18n"
+        end
+        File.write("config/database.yml", database_config.to_yaml)
+      end
+
+      def manageiq_pot_filename
+        "locale/manageiq.pot"
+      end
+
+      def manageiq_pot_file_needs_commit?
+        needs_commit = false
+        diffs = git_diff_stats
+        return false if diffs.length != 1
+        return false unless diffs.key?(manageiq_pot_filename)
+        stats = diffs[manageiq_pot_filename]
+        puts "+#{stats[:insertions]}, -#{stats[:deletions]} for #{manageiq_pot_filename}"
+          # If the only changes are to POT-Creation-Date and PO-Revision-Date,
+          # then there are no substantive changes
+          # Hence why this code checks for precisely 2 insertions and deletions
+        (stats[:insertions] != 2) || (stats[:deletions] != 2)
+      end
+
+      def git_diff_stats
+        manageiq_repo.git.capturing.diff(:numstat => true).split("\n").each_with_object({}) do |diff, stats|
+          insertions, deletions, fname = diff.split("\s")
+          stats[fname] = {:insertions => insertions.to_i, :deletions => deletions.to_i }
+        end
+      end
+
+      def manageiq_repo_path
+        manageiq_repo.path || File.join(__dir__, "../../../repos/ManageIQ/manageiq")
+      end
+
+      def manageiq_repo
+        @manageiq_repo ||= Repo.new('manageiq')
```

Comment:
Minor, but can you change this to `"ManageIQ/manageiq"`?  The reason I'm asking is because in a separate PR I will be doing some extraction/refactoring, and I'm changing the repo interface to enforce an `org/repo` naming.  org/repo should still work in the current master version.